yt/data_objects/index_subobjects/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .grid_patch import AMRGridPatch
```

Comment:
Is this required? It seems the only place it is imported from is in `yt/data_objects/api.py` above.